crates/iota/src/name_commands.rs

```diff
@@ -544,6 +560,36 @@ impl NameCommand {
                     .await?,
                 )
             }
+            Self::UnsetTargetAddress { domain, opts } => {
+                let entry = get_registry_entry(&domain, &iota_client).await?;
+                if entry.name_record.target_address.is_none() {
+                    anyhow::bail!("target address is already unset");
+                }
+
+                let nft_id = get_owned_nft_by_name::<IotaNamesRegistration>(&domain, context)
+                    .await?
+                    .id();
```

Comment:
Any reason to not allow this also with subdomains?

Reply:
unsupported in IOTA-Names? Because you're just adding this command here now?

Reply:
Haha, I actually just forgot that I added this call 🤦‍♀️